systems/WikibaseRepo/05-Building_Block_View.md

```diff
@@ -81,7 +81,7 @@ TBA
 | ------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | SpecialWikibasePage                         | base class          | Base class for Wikibase Special Pages                                                                                                                                                                        |
 | EntityIdParser                              | Handling user input | Interface for objects that can parse strings into EntityIds                                                                                                                                                  |
-| TokenCheckInteractor                        | Handling user input | Checks a submitted edit token                                                                                                                                                                                 |
+| TokenCheckInteractor                        | Handling user input | Checks a submitted edit token                                                                                                                                                                                |
```

Comment:
Not sure why this line is showing up in the diff?
This is the second PR i have seen this line apparently be touched in now but I can't actually see a change!

Reply:
There is a space that gets removed. It happens on the pre-commit hook. I didn't notice it sneaked in, but anyway it has to be fixed or it will keep annoying us